packages/kyt-core/src/actions/__tests__/dev.test.js

```diff
@@ -37,45 +24,19 @@ const dev = require('../dev');
 describe('dev', () => {
   // this can be replaced with jest.clearAllMocks in jest 16.0.0
   beforeEach(() => {
-    [
-      chokidar,
-      chokidar.on,
-      chokidar.watch,
-      nodemon,
-      nodemon.on,
-      nodemon.once,
-      devMiddleware,
-      express,
-      express.use,
-      express.listen,
-      express.static,
-      ifPortIsFreeDo,
-      webpackCompiler,
-      buildConfigs,
-    ].forEach(mock => mock.mockClear());
+    jest.clearAllMocks();
   });
 
   const href = { href: 'href' };
-  const port = 1000;
+  const port = 1024;
```

Comment:
I know it doesn't matter in this test, but just curious why the port change here? Double check that the test fails or something?

Reply:
the port check is not mocked, and I think 1000 is invalid